### DISCUSSION TITLE
Doubt

### DISCUSSION THREAD

**Discussion by UjjwalSainiCDC** [UjjwalSaini07/VedicSadhana#67] [Q&A] [answered]
I wont able to search Vedic Vani Online How i found that

**Comment by UjjwalSaini07** [comment, marked as answer]
HIii UjjwalCDC !!
I'm the Also Ujjwal Repo Owner.
I deploy my website on versel app u find in repo iprovide u all links here.

**_Website Link -_** https://vedic-sadhana.vercel.app/
**_Repo Link -_** https://github.com/UjjwalSaini07/VedicSadhana

If u Want to sponsor me so wait one more day i switch the toggle for sponsoring to my project so that i do increase my effort on this project and i get more motivation to create projects like this.

Thanx a lot for choosing Discussion

**Comment by UjjwalSaini07** [comment]
Thank u for providing Answer as selected
